lib/module.js

```diff
@@ -72,23 +72,34 @@ function Typo3 (options) {
 
   // add components
   // you can disable it if you don't want them in build
-  // you can import them in your source code
-  if (options.components) {
-    this.addPlugin({
-      src: resolve(__dirname, './plugins/components.js'),
-      options,
-      fileName: join('typo3', 'plugins.components.js')
-    })
-  }
+  this.nuxt.hook('components:dirs', (dirs) => {
```

Comment:
This is great functionality, but we can add it only when user has `components:true` in their `nuxt.config` - It's not enabled by default. 

https://nuxtjs.org/docs/configuration-glossary/configuration-components 

First of all, we have to check if user has this option enabled - if yes we can use the hook, but in another case we have to keep fallback. So I would rather keep plugins/components and forms files to register them globally if user does not use components auto import feature.

I know there is also possibility to override this option from module, but I would rather not do this without user awareness. 

What do you think ?